modules/xmpp/moderator.js

```diff
@@ -210,11 +210,12 @@ Moderator.prototype.createConferenceIq = function() {
                 value: this.options.conference.minBitrate
             }).up();
     }
-    if (this.options.conference.openSctp !== undefined) {
+    if (this.options.conference.openBridgeChannel !== undefined) {
```

Comment:
we only want to do this if we are going to use the datachannel. This is basically telling the JVB "hey, open a datachannel" so it's a boolean.